TypeScript.YAML-tmLanguage

```diff
@@ -653,7 +653,8 @@ repository:
 
   comment-line:
     name: comment.line.ts
-    match: (//).*$\n?
+    begin: //
+    end: $
```

Comment:
I tested it on Mac OS X (El Capitan 10.11.6), Windows 8.1 (64 Bit) and Linux (14.04.5 LTS) and $ is sufficient on all platforms.

I illustrated the problem in these two screenshots:
old grammar:
![old](https://cloud.githubusercontent.com/assets/282412/17923295/d1d78c3a-69e3-11e6-9fe2-4431ef96d3d2.png)
new grammar:
![new](https://cloud.githubusercontent.com/assets/282412/17923300/d690a8e2-69e3-11e6-97da-a57a9c3c5c12.png)


Editors like [Atom](https://atom.io/) are currently not able to highlight TODO line comments like this:

`// TODO: enable highlighting of line comments`

only block comment highlighting does work:

`/* TODO: this will be highlighted */`

That’s because the file

https://github.com/TypeStrong/atom-typescript/blob/master/scripts/grammar.js#L113

references these two files:

https://raw.githubusercontent.com/Microsoft/TypeScript-TmLanguage/master/TypeScript.YAML-tmLanguage
https://raw.githubusercontent.com/Microsoft/TypeScript-TmLanguage/master/TypeScriptReact.YAML-tmLanguage

See also this issue: https://github.com/TypeStrong/atom-typescript/pull/1035